lib/server/utilities/findAndCountAllToHttpView.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef CountedData
+ * @property {Number} count
+ * @property {Object[]} rows
+ */
+
+/**
+ * @typedef CountedDataView
+ * @property {{totalCount: Number, pageCount: Number}} meta
+ * @property {Object[]} data
+ */
+
+/**
+ * Adapt services and usecases results of type 'findAndCountAll' to api
+ * @param {CountedData} data count from findAndCountAll method
+ * @returns {CountedDataView} adapted result
+ */
+function findAndCountAllToHttpView({ count, rows }) {
```

Comment:
The idea is good, but why do you mention services and usecases here? What you do is format `CountedData` to go through HTTP API. I would name it `countedDataToHttpView`.
You don't have to mention `findAndCountAll` because you don't care where the data come from, the only thing you care about is that it respect the `CountedData` structure.
(If you rename `CountedData` to `CountedRows` as I suggested, please update the naming here accordingly)

Reply:
yes refactored, but ... as it's not belong to scope of this PR I will add in another one